python/pyspark/sql/connect/client/core.py

```diff
@@ -1013,7 +1060,9 @@ def explain_string(self, plan: pb2.Plan, explain_mode: str = "extended") -> str:
         Return explain string for given plan.
         """
         if logger.isEnabledFor(logging.INFO):
-            logger.info(f"Explain (mode={explain_mode}) for plan {self._proto_to_string(plan)}")
+            logger.info(
+                f"Explain (mode={explain_mode}) for plan {self._proto_to_string(plan, True)}"
```

Comment:
Is it possible for debugging to get the full log?

Reply:
it may produce massive loggings, which is mostly the sparkplans in binary format.
I guess it won't hurt debuggability too much since it only truncates large bytes/string